packages/react/src/components/Tabs/next/Tabs.stories.js

```diff
@@ -29,44 +29,87 @@ export default {
   },
 };
 
+const tabPanelEnums = {
```

Comment:
Small nit, but I wouldn't be opposed to a shorter/simpler name here

```suggestion
const tabs = {
```

Or if you'd like, could even break these out into their own variables to cut down on the chaining by one level
```
const tab1 = { labelText: '', panelText: '' };
const tab2 = { ... };
...etc
```